feature-libs/order/assets/translations/en/order.i18n.ts

```diff
@@ -240,4 +240,16 @@ export const MyAccountV2Order = {
     viewAllOrders: 'View All Orders',
     noInvoices: 'Invoices are not generated yet. Please come back later.',
   },
+  myAccountView: {
+    item: '{{value}} Item',
+    items: '{{value}} Items',
+    heading: 'Orders And Returns',
+    orderNumber: 'Order Number ({{value}})',
+    purchasedOn: 'Purchased On: {{value}}',
+    orderedItems: 'Ordered Items: {{value}}',
+    totalPrice: 'Total Price: {{value}}',
+    orderDetails: 'Order Details',
+    returnOrder: 'Return Order',
+    showMore: 'Show More',
+  },
```

Comment:
[MEDIUM]
All those labels seem to be related to only the _order_ listing in the My Account landing page.
I have problem to understand why we're binding the very-general-sounding i18n namespace `myAccountView.xxx` to the (lazy-loadable) translation chunk `myAccountV2Order.json` (which is meant to contain only order-feature-related in the redesigned my account pages stuff).

Did you consider to name this main translation key `myAccountViewOrder` or `myAccountOrder` instead?

Reply:
done

Reply:
I have problem to find the code change in the latest version of this PR. I can still see `myAccountView` instead of `myAccountViewOrder`.